devicetypes/smartthings/hue-bulb.src/hue-bulb.groovy

```diff
@@ -167,6 +168,10 @@ void setColorTemperature(value) {
     }
 }
 
+void installed() {
+	// setup device watch
+	sendEvent(name: "checkInterval", value: 60 * 6, displayed: false)
```

Comment:
for now the agreed interval for switches is 30 minutes. is the 6 minutes for testing?